plugins/user-settings-backend/api-report-alpha.md

```diff
@@ -5,7 +5,7 @@
 ```ts
 import { BackendFeature } from '@backstage/backend-plugin-api';
 
-// @alpha
+// @alpha @deprecated
```

Comment:
I don't think that this deprecation notice will show up anywhere as we're using `import(...)`.

Thinking that it might be worth just removing this import from alpha instead, and making the breaking change to the `alpha` exports so that it doesn't export this anymore.